### ISSUE TITLE
question about SPM and BPE

### ISSUE THREAD

**Issue by jemesome** [Helsinki-NLP/OPUS-MT-train#104]
Noob here, running the option2 using docker to install the tornado web server with several language models as listed in the  [readme here](https://github.com/Helsinki-NLP/Opus-MT). I ran the dockerfile after cloning from [here ](https://github.com/Helsinki-NLP/Opus-MT).  

What is the difference between models in  [OPUS-MT-train](https://github.com/Helsinki-NLP/OPUS-MT-train) and models in the [Tatoeba-Challenge](https://github.com/Helsinki-NLP/Tatoeba-Challenge) ? Looking at the benchmark on test set for for the Spanish to english model, the [ES-EN model](https://github.com/Helsinki-NLP/OPUS-MT-train/tree/master/models/es-en) trained on tatoeba dataset has lower metric compared to [the same model in Tatoeba- challenge (Spa-eng)](https://github.com/Helsinki-NLP/Tatoeba-Challenge/tree/master/models/spa-eng).

are these 2 the same model trained on different dataset? is the model based on Tatoeba newer and better compared to the one in OPUS-MT-train as suggested by the benchmark metric?

**Comment by jorgtied** [comment]
Most of the Tatoeba models are better than the earlier opus-mt models. The training data is a bit cleaner, sometimes also a bit bigger. Models are also sometimes bigger than the original OPUS-MT models (transformer-big instead of transformer-base). But there are different variants and you need to check the model card / README.
